modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -34,4 +36,74 @@ export default class NestedAccordion extends Base {
 			$items[ index ].appendChild( element );
 		} );
 	}
+
+	applyDefaultStateCondition() {
+		if ( ! this.elements ) {
+			return;
+		}
+
+		const accordionItems = this.elements.$items,
+			defaultState = this.getDefaultStateCondition(),
+			stateFirstExpanded = 'first_expended';
```

Comment:
We can add this const to the `getDefaultSettings()` just as it was done in the [nested-tabs](https://github.com/elementor/elementor/blob/1b13505cf2a150861e4621c093a43c8a4de0f784/modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js#L38)